plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueToTrinoConverter.java

```diff
@@ -93,14 +94,19 @@ public static Table convertTable(com.amazonaws.services.glue.model.Table glueTab
         return tableBuilder.build();
     }
 
-    private static Column convertColumn(com.amazonaws.services.glue.model.Column glueColumn)
+    private static Column convertColumn(com.amazonaws.services.glue.model.Column glueColumn, String serDe)
     {
-        return new Column(glueColumn.getName(), HiveType.valueOf(glueColumn.getType().toLowerCase(Locale.ENGLISH)), Optional.ofNullable(glueColumn.getComment()));
+        if (HiveStorageFormat.CSV.getSerDe().equals(serDe)) {
+            return new Column(glueColumn.getName(), HiveType.HIVE_STRING, Optional.ofNullable(glueColumn.getComment()));
```

Comment:
please add a comment here with short explanation why we need to do that.

Reply:
done